llvm/lib/Target/AMDGPU/AMDGPUAsmPrinter.cpp

```diff
@@ -134,6 +135,12 @@ void AMDGPUAsmPrinter::initTargetStreamer(Module &M) {
     getTargetStreamer()->getPALMetadata()->readFromIR(M);
 }
 
+uint64_t AMDGPUAsmPrinter::getMCExprValue(const MCExpr *Value) {
+  int64_t Val;
+  Value->evaluateAsAbsolute(Val);
```

Comment:
unchecked result

Reply:
As with the other MCExpr value retrieval helper in this PR: now errors if it cannot resolve; aim to remove this helper when consumers of `SIProgramInfo` allow MCExprs directly.